src/ntwi.cpp

```diff
@@ -1,12 +1,13 @@
 #include "dataset.hpp"
 #include "fcm.hpp"
 #include <random>
+#include "knnImp.hpp"
```

Comment:
I teraz razem z tym nagłówkiem dostajemy cały `namespace std` w prezencie. To jest **bardzo** zła praktyka

Reply:
done